ge/templates/ge/edit_fund.html

```diff
@@ -15,7 +15,11 @@
   <table class="data-form">
     {{ form.as_table }}
   </table>
-  <button type="submit" id="edit_submit" >Save changes</button>
+  <button type="submit" id="edit_submit">Save changes</button>
 </form>
 
+<form id="delete_fund" action="{% url 'delete_fund' form.instance.pk %}" method="post">
+  {% csrf_token %}
+  <button type="submit" id="delete_submit">Delete this fund</button>
```

Comment:
@ztucker4 Please add a bit of javascript to confirm deletion, as that's the norm and a mis-click could be bad.  It can be as simple as what I did in [the link shortener app](https://github.com/UCLALibrary/link-shortener/blob/main/shortlinks/templates/shortlinks/show_links.html#L30) - just an `onclick` in the button, and the same [javascript I used](https://github.com/UCLALibrary/link-shortener/blob/main/static/js/main.js) (with the comment and text changed); just add the JS file to `/static/js/`.

It could even be as simple as adding `onclick="return confirm('Delete this fund?');"`, but I prefer JS and CSS both to be external, not inline.